### PR TITLE
fix: get async ele

### PR DESCRIPTION
**Pull request by tomerlichtash**
## Related Issues

Fixes <link_to_github_issue>

## Related PRs

| branch       | PR         |
| ------------ | ---------- |
| service a PR | Link to PR |
| service b PR | Link to PR |

## Description

A few sentences describing the overall goals of the pull request's commits.

## Must

- [ ] Tests
- [ ] Documentation (if applicable)

### CODE REVIEW

File: packages/user-management-widget/src/lib/mixins/formMixin.ts [hunk 1]
```diff
@@ -46,6 +46,7 @@ export const formMixin = createSingletonMixin(
         this.getFormInputs(rootEle).forEach((input) => {
           // eslint-disable-next-line no-param-reassign
           input.value = '';
+          input.removeAttribute('invalid');
```
Comment (nirgur) [inline packages/user-management-widget/src/lib/mixins/formMixin.ts:49]: I don't think we should change the components invalid attribute from the driver
Why do we need it?

File: packages/user-management-widget/src/lib/widget/drivers/BaseDriver.ts [hunk 2]
```diff
@@ -3,6 +3,21 @@ import { Logger } from '../../mixins/loggerMixin/types';
 type Empty = null | undefined;
 type RefOrRefFn = Element | (() => HTMLElement | Empty) | Empty;
 
+const waitForElement = async (ele: RefOrRefFn, timeout: number) =>
```
Comment (nirgur) [inline packages/user-management-widget/src/lib/widget/drivers/BaseDriver.ts:6]: Helper file?

**Review by nirgur — COMMENTED** [review]
👍 
Added 2 comments

File: packages/user-management-widget/src/lib/mixins/formMixin.ts (hunk 1 above)
Comment (tomerlichtash) [reply packages/user-management-widget/src/lib/mixins/formMixin.ts:49]: We reset the form, but the validations states remains dirty. This is intended to clear the validation state as well.

File: packages/user-management-widget/src/lib/widget/drivers/BaseDriver.ts (hunk 2 above)
Comment (tomerlichtash) [reply packages/user-management-widget/src/lib/widget/drivers/BaseDriver.ts:6]: Done. Also moved types to a separate file to avoid circular dependency.

**Review by nirgur — APPROVED** [review]

**Pull request merged**